test/e2e/action_test.go

```diff
@@ -424,13 +549,13 @@ func getTypeInstanceInputForPolicy() *hublocalgraphql.CreateTypeInstanceInput {
 	}
 }
 
-func getTypeInstanceInputForDownload(testValue string) *hublocalgraphql.CreateTypeInstanceInput {
+func getTypeInstanceInputForDownload(testValues map[string]interface{}) *hublocalgraphql.CreateTypeInstanceInput {
 	return &hublocalgraphql.CreateTypeInstanceInput{
 		TypeRef: &hublocalgraphql.TypeInstanceTypeReferenceInput{
 			Path:     "cap.type.capactio.capact.validation.download",
 			Revision: "0.1.0",
 		},
-		Value: map[string]interface{}{"key": testValue},
+		Value: testValues,
```

Comment:
wow 😄  it's totally against our contract. It should be:
```go
		Value: testValues,
		Backend: &hublocalgraphql.TypeInstanceBackendInput{
			ID: "123", // storage backend ID
			Context: map[string]interface{}{
				"provider": "dotenv",
			},
		},
```

here is our contract:


- Mutation
  ```graphql
  mutation CreateTypeInstances {
    createTypeInstances(
      in: {
       typeInstances: [
        {
          alias: "helm-release"
          typeRef: { path: "cap.type.helm.chart.release", revision: "0.1.0" }
          value: {
            key: "test" # same as it was
          }
          backend: {
            id: "123" # it's like that already.
            context: { # new property of type `Any!`
              name: "release-name",
              namespace: "release-namespace",
            }
          }
        }
      ]
    }
    ) {
      id
      alias
    }
  }
  ```
- Query
  ```graphql
  query GetTypeInstance($typeInstanceID: ID!) {
    typeInstance(id: $typeInstanceID) {
      id
      typeRef {
        path
        revision
      }
      backend { # current backend entry
        id
        abstract
      }    
      lockedBy
      latestResourceVersion {
          resourceVersion
          createdBy
          metadata {
            attributes {
              path
              revision
            }
          }
          spec {
            value
          }
          backend {
            context { # new property of type `Any!`
              name
              namespace
            }
          }
      }
    }
  }
  ```
Argo actions need to support that, the `backend` needs to fetch for the given TypeInstance. Let's dicsuss that f2f.

Reply:
discussed f2f :)